chain/postgresdb/pgdb/pgdb.go

```diff
@@ -87,22 +94,29 @@ func (p *Store) Len(ctx context.Context) (int, error) {
 	FROM
 		beacon_details
 	WHERE
-		beacon_id = :id`
+		beacon_id = :beacon_id`
 
 	data := struct {
-		ID int `db:"id"`
+		BeaconID int64 `db:"beacon_id"`
```

Comment:
Why the change to int64? Seems unnecessary for the `beacon_id`.

Reply:
Good point. I wanted to make sure that there's no type mismatch across 32 and 64 bit systems, but it should be an issue. I don't think we'll overflow beacon_id on 32 bit any time soon. Reverted.